--- a/rules/evdev.xml.in	2016-10-09 06:10:38.746286456 +0200
+++ b/rules/evdev.xml.in	2016-10-09 06:11:04.917143614 +0200
@@ -6427,6 +6427,12 @@
       </option>
       <option>
         <configItem>
+          <name>caps:f13</name>
+          <_description>Caps Lock is F13 key</_description>
+        </configItem>
+      </option>
+      <option>
+        <configItem>
           <name>caps:ctrl_modifier</name>
           <_description>Make Caps Lock an additional Ctrl</_description>
         </configItem>
